README.md

```diff
@@ -1,5 +1,7 @@
 # SymbiFlow Architecture Definitions
 
+**If you are looking for instructions how to use SymbiFlow, take a look at the [symbiflow-examples repository](https://github.com/SymbiFlow/symbiflow-examples).**
```

Comment:
I think this needs to be more along the lines of "This repository is used during the development of architecture support in SymbiFlow, if you are looking to use the toolchain you should start at symbiflow-examples repository".

Reply:
OK, done!